modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -4,11 +4,22 @@
 	--n-accordion-border-width: 1px;
 	--n-accordion-border-color: #{$gray-light};
 	--n-accordion-border-style: solid;
+  	--n-accordion-item-title-space-between: 0px;
+  	--n-accordion-item-title-distance-from-content: 0px;
+  	--n-accordion-padding:0px;
+  	--n-accordion-border-radius :0px;
 
 	.e-n-accordion {
 
 		&-item {
 
+		  padding: var(--n-accordion-padding) ;
+		  border-radius:  var(--n-accordion-border-radius);
+
+		  &:not(:last-child) {
+				margin-bottom: var(--n-accordion-item-title-space-between);
+		  	}
+
 			> .e-con {
 				border-width: var(--n-accordion-border-width);
 				border-color: var(--n-accordion-border-color);
```

Comment:
What is this styling used for? Are these not defined by the php controls? Is this pure as default values?

Reply:
yes
it will be overridden in the content styling PR